swagger_to_uml.py

```diff
@@ -195,6 +195,11 @@ def from_dict(name, d):
                 required=property_name in d.get('required', [])
             ))
 
+        try:
```

Comment:
I think the `except Exception` is too broad here. Also, the error message is not really helpful. What about something like

```py
if not 'type'. in d:
  print('required key "type" not found in dictionary ' + json.dumps(d), file=sys.stderr)
```